opensearch-operator/opensearch-gateway/services/os_client.go

```diff
@@ -307,6 +306,68 @@ func (client *OsClusterClient) DeleteSecurityResource(ctx context.Context, resou
 	return doHTTPDelete(ctx, client.client, path)
 }
 
+// GetISMConfig performs an HTTP GET request to OS to get the ISM policy resource specified by name
+func (client *OsClusterClient) GetISMConfig(ctx context.Context, resource, name string) (*opensearchapi.Response, error) {
```

Comment:
Since `resource` is always `_ism` I think it is better to hardcode it here in the client and not have it as a parameter.